python/pyarrow/conftest.py

```diff
@@ -54,6 +55,7 @@
 
 defaults = {
     'acero': False,
+    'azure': False,
```

Comment:
Do we want to enable them when the filesystem can be imported? (like is done for s3 around line 156 below)

Reply:
Yes, I missed this.

Reply:
Done